src/assets/locales/en.js

```diff
@@ -167,6 +167,20 @@ export default {
           duplicate: "Duplicate",
           analyse: "Analyse",
           share: "Share",
+          delete: "Delete",
+        },
+        dialog: {
+          delete: {
+            title: "Are you sure you want to delete this plio?",
+            description:
+              "This will permananently delete this plio and all the data associated with it, including all the responses. Once deleted, the data cannot be covered.",
+          },
+        },
+        toast: {
+          delete: {
+            success: "Plio deleted successfully",
+            error: "There was an error. Please try again!",
```

Comment:
I thought that since this toast will appear only after the delete action, mentioning the delete action again is redundant.

Reply:
I strongly think we should add that. Because we also have an error toast that just says "Something went wrong. Please try again"

And to handle the incorrect OTP error, we have written "Incorrect OTP entered. Please try again!"

Also because any kind of 400 or 500 error throws a generic please try again toast, it would be better to explicitly tell them what went wrong. I hate it when in apps or website toasts come up that show a generic message